src/client-configuration.ts

```diff
@@ -62,6 +62,11 @@ export interface ClientConfiguration {
    * Must match format: https://www.w3.org/TR/trace-context/#traceparent-header
    */
   traceparent?: string;
+  /**
+   * Enable or disable typechecking of the query before evaluation. If no value
+   * is provided, the database defaults using typechecking.
```

Comment:
I think it defaults to using the database's setting, so let's reword a bit. https://github.com/fauna/core/blob/master/ext/api/src/main/resources/openapi.yml#L73-L75

```suggestion
   * Enable or disable typechecking of the query before evaluation. If no value
   * is provided, the `typechecked` setting on the database will be used.
```